test/spec/ol/interaction/mousewheelzoom.test.js

```diff
@@ -62,110 +62,106 @@ describe('ol.interaction.MouseWheelZoom', function() {
 
   describe('handleEvent()', function() {
 
-    it('works on Firefox in DOM_DELTA_PIXEL mode (trackpad)', function(done) {
-      const origHasFirefox = _ol_has_.FIREFOX;
-      _ol_has_.FIREFOX = true;
-      map.once('postrender', function() {
-        expect(interaction.mode_).to.be(MouseWheelZoom.Mode_.TRACKPAD);
-        _ol_has_.FIREFOX = origHasFirefox;
-        done();
-      });
-      const event = new MapBrowserEvent('wheel', map, {
-        type: 'wheel',
-        deltaMode: WheelEvent.DOM_DELTA_PIXEL,
-        deltaY: _ol_has_.DEVICE_PIXEL_RATIO,
-        target: map.getViewport(),
-        preventDefault: Event.prototype.preventDefault
-      });
-      event.coordinate = [0, 0];
-      map.handleMapBrowserEvent(event);
-    });
-
-    it('works in DOM_DELTA_PIXEL mode (trackpad)', function(done) {
-      const origHasFirefox = _ol_has_.FIREFOX;
-      _ol_has_.FIREFOX = false;
-      map.once('postrender', function() {
-        expect(interaction.mode_).to.be(MouseWheelZoom.Mode_.TRACKPAD);
-        _ol_has_.FIREFOX = origHasFirefox;
-        done();
-      });
-      const event = new MapBrowserEvent('wheel', map, {
-        type: 'wheel',
-        deltaMode: WheelEvent.DOM_DELTA_PIXEL,
-        deltaY: 1,
-        target: map.getViewport(),
-        preventDefault: Event.prototype.preventDefault
-      });
-      event.coordinate = [0, 0];
-      map.handleMapBrowserEvent(event);
-    });
-
-    describe('spying on ol.interaction.Interaction.zoomByDelta', function() {
-      beforeEach(function() {
-        sinon.spy(Interaction, 'zoomByDelta');
-      });
-      afterEach(function() {
-        Interaction.zoomByDelta.restore();
-      });
-
-      it('works in DOM_DELTA_LINE mode (wheel)', function(done) {
+    if (FIREFOX) {
+      it('works on Firefox in DOM_DELTA_PIXEL mode (trackpad)', function(done) {
         map.once('postrender', function() {
-          const call = Interaction.zoomByDelta.getCall(0);
-          expect(call.args[1]).to.be(-1);
-          expect(call.args[2]).to.eql([0, 0]);
+          expect(interaction.mode_).to.be(MouseWheelZoom.Mode_.TRACKPAD);
           done();
         });
         const event = new MapBrowserEvent('wheel', map, {
           type: 'wheel',
-          deltaMode: WheelEvent.DOM_DELTA_LINE,
-          deltaY: 3.714599609375,
+          deltaMode: WheelEvent.DOM_DELTA_PIXEL,
+          deltaY: DEVICE_PIXEL_RATIO,
           target: map.getViewport(),
           preventDefault: Event.prototype.preventDefault
         });
         event.coordinate = [0, 0];
         map.handleMapBrowserEvent(event);
       });
+    }
 
-      it('works on Safari (wheel)', function(done) {
-        const origHasSafari = _ol_has_.SAFARI;
-        _ol_has_.SAFARI = true;
+    if (!FIREFOX) {
```

Comment:
Isn't this too broad? Should this test for Safari explicitly?

Reply:
This diff view may be making it hard to see, but this condition wraps a test that used to set `FIREFOX = false`.  See https://github.com/openlayers/openlayers/pull/7764/files#diff-7c3aa34c433b5b7112b6f12d7aeea7aaL86.